### ISSUE TITLE
Warning happening on HA Supervised Installations

### ISSUE THREAD

**Issue by sca075** [sca075/valetudo_vacuum_camera#53]
### Checklist

- [X] I have updated the integration to the latest version available
- [X] I have checked if the problem is already reported

### The problem

Warning in Home Assistant for detected blocking point..

### What version of an integration has described problem?

v1.5.0

### What was the last working version of an integration?

v1.4.9

### What vacuum model do you have problems with?

Roborock.V1

### Please firmware installed on your Vacuum.

2023.10.0

### What version of Home Assistant do you use?

2023.12.1

### What type of installation are you running?

Home Assistant Supervised

### Camera's settings.

```yaml
Default settings 180 rotation and 50% crop.
```

### Logs or Errors shown in the HA snapshots (please enable the Debug Mode)

```shell
2023-12-11 08:03:51.808 INFO (MainThread) [custom_components.valetudo_vacuum_camera.utils.draweble] Drawing silenttepidstinkbug Robot With Angle: 165
2023-12-11 08:03:51.809 DEBUG (MainThread) [custom_components.valetudo_vacuum_camera.valetudo.image_handler] silenttepidstinkbug Image Cropping:25 Image Rotate:180
2023-12-11 08:03:51.809 DEBUG (MainThread) [custom_components.valetudo_vacuum_camera.valetudo.image_handler] Image original size: 5120, 5120
2023-12-11 08:03:51.809 DEBUG (MainThread) [custom_components.valetudo_vacuum_camera.valetudo.image_handler] Image size after first crop: 2560, 2560
2023-12-11 08:03:51.809 DEBUG (MainThread) [custom_components.valetudo_vacuum_camera.valetudo.image_handler] Auto Crop and Trim Box data: (1896, 2006, 3250, 2935)
2023-12-11 08:03:51.810 DEBUG (MainThread) [custom_components.valetudo_vacuum_camera.valetudo.image_handler] Auto Crop and Trim image size: (1354, 929)
2023-12-11 08:03:51.883 DEBUG (MainThread) [custom_components.valetudo_vacuum_camera.camera] Applied silenttepidstinkbug image rotation: 180
2023-12-11 08:03:51.931 INFO (MainThread) [custom_components.valetudo_vacuum_camera.camera] silenttepidstinkbug: Image update complete
2023-12-11 08:03:51.931 DEBUG (MainThread) [custom_components.valetudo_vacuum_camera.camera] Adjusted silenttepidstinkbug: Frame interval: 0.43572
2023-12-11 08:03:51.935 WARNING (MainThread) [homeassistant.util.async_] Detected blocking call to sleep inside the event loop by custom integration 'valetudo_vacuum_camera' at custom_components/valetudo_vacuum_camera/camera.py, line 597: f"{proc_insp.PsutilWrapper().psutil.cpu_percent(interval=self.frame_interval)}"), please create a bug report at https://github.com/sca075/valetudo_vacuum_camera/issues
2023-12-11 08:03:52.936 DEBUG (MainThread) [custom_components.valetudo_vacuum_camera.camera] silenttepidstinkbug Camera CPU usage %: 0.2
2023-12-11 08:03:52.937 DEBUG (MainThread) [custom_components.valetudo_vacuum_camera.camera] silenttepidstinkbug Camera Virtual Memory usage %: 25.0
2023-12-11 08:03:52.949 INFO (MainThread) [custom_components.valetudo_vacuum_camera.valetudo.MQTT.connector] Received valetudo/SilentTepidStinkbug image data from MQTT
```

### Additional information

the interesting thing is.. well CPU is stable at 3% while processing the image.. so there is an improvement for sure but we need to understand why on Home Assistant OS this not happening?

**Comment by sca075** [comment]
Tests done on HA OS and supervised (debian) confirms the following:
- psutil_home_assistant:
- On HA OS is possible to run psutil_home_assistant.PsutilWrapper().psutil.cpu_percent(interval=self._frame_interval) ->  no issue
- On Supervised not on OS, the same command is detected as blocking, from that psutil docs, this is a blocking call -> it is because psutil is using the running thread. psutil.cpu_percent(interval=None) should be the correct way to implement it.
- Either the cases.. This is really helping to warranty the access to the thread in HA asyinc mode so we will simply run it in background.

**Comment by sca075** [comment]
Released v1.5.1 including the above fixes.
